install_and_configure.yml

```diff
@@ -0,0 +1,121 @@
+---
+ - hosts: epidose
+   gather_facts: false
+   become: yes
+   vars:
+     ssh_key: '~/.ssh/id_rsa.pub'
+
+   tasks:
+    - name: create 'testgroup' group
+      tags: production, development
+      group:
+        name: testgroup
+        state: present
+
+    - name: allow 'testgroup' group to have passwordless sudo
+      tags: production, development
+      lineinfile:
+        dest: /etc/sudoers
+        state: present
+        regexp: '^%testgroup'
+        line: '%testgroup ALL=(ALL) NOPASSWD: ALL'
+        validate: 'visudo -cf %s'
+  
+    - name: add sudo user to 'testgroup' group
+      tags: production, development
+      user:
+        name: epidose
+        groups: testgroup
+        append: yes
+        state: present
+        createhome: yes
+
+    - name: deploy ssh key
+      tags: production, development
+      authorized_key:
+        key: "{{ lookup('file', ssh_key) }}"
+        user: epidose
+
+    - name: install libraries and packages
+      tags: production, development
+      become: yes
+      become_user: epidose
```

Comment:
When is the user `epidose` created?

Reply:
Just before installing the libraries, exactly in this part:

```- name: add sudo user to 'testgroup' group 
      tags: production, development
      user:
        name: epidose
        groups: testgroup
        append: yes
        state: present
        createhome: yes

Reply:
Thanks!  Why in `testgroup`, rather than `epidose`?

Reply:
There is no particular reason.
I will change it to epidose.